tests/network/bindingplugin_slirp.go

```diff
@@ -33,7 +33,8 @@ import (
 	"kubevirt.io/client-go/log"
 
 	"kubevirt.io/kubevirt/pkg/libvmi"
-	virtconfig "kubevirt.io/kubevirt/pkg/virt-config"
+	"kubevirt.io/kubevirt/pkg/virt-config/featuregate"
+
```

Comment:
nit: remove newline

Reply:
The newline separates production from e2e tests packages.